src/App.tsx

```diff
@@ -17,18 +49,30 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter filtredTodos={filtredTodos} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isLoading
```

Comment:
Why `!IsLoading` here? This is confusing because when `isLoading` === `true` you render the `TodoList` when it should be the other way around.